wiris/moodle-atto_wiris
# Fix KB-5540: formulas dissapear from the editor when providing feedback to an assignment

This change fixes bug KB-5540, avoiding double parsing inserted formulas so they don't dissapear from the editor's textarea when providing feedback to an assignment.

**How to reproduce**

- Create an assigment with the teacher's role
- Answer the assignment with the student's role
- Provide feedback with the teacher's role
 -Add a formula using the atto_editor's MathType button
- Click on the feedback button
- Accept changes on the modal window that appears

**Expected result**

The formula is added in both comment editor's target element on the sidebar and the assignment page on the main side.

**Actual result**

The formula dissapears.
